corehq/apps/smsbillables/generator.py

```diff
@@ -153,8 +153,9 @@ def arbitrary_backend_ids():
     for backend in get_available_backends().values():
         backend_instance = data_gen.arbitrary_unique_name("back")
         backend_ids[backend.get_api_id()] = backend_instance
-        sms_backend = SMSBackend()
+        sms_backend = backend()
```

Comment:
@nickpell The smsbillables tests were failing with my changes in this PR because this code was using the `SMSBackend` superclass and there are now unimplemented methods there that raised exceptions. So I just updated this to use the actual subclass instead and now the tests pass. It seems like this would be fine, but let me know if you think that's a bad idea for any reason.